package.json

```diff
@@ -14,35 +14,19 @@
   },
   "scripts": {
     "build": "sh build.sh",
-    "clean": "lerna run clean --parallel --no-bail",
+    "clean": "git clean -Xdf --exclude=\"!.env\"",
```

Comment:
Why not add `.env` to the gitignore?

Reply:
It is in the `.gitignore` -- this is saying, do not delete the `.env` when doing a clean. Usually it has secrets you would prefer to keep